pkg/util/overridemanager/overridemanager_test.go

```diff
@@ -14,6 +19,195 @@ import (
 	"github.com/karmada-io/karmada/test/helper"
 )
 
+func Test_overrideManagerImpl_ApplyOverridePolicies(t *testing.T) {
+	deployment := helper.NewDeployment(metav1.NamespaceDefault, "test1")
+	deployment.Labels = map[string]string{
+		"foo": "foo",
+	}
+	deploymentObj, _ := utilhelper.ToUnstructured(deployment)
+
+	clusterRole := helper.NewClusterRole("test2", nil)
+	clusterRole.Labels = map[string]string{
+		"bar": "bar",
+	}
+	clusterRoleObj, _ := utilhelper.ToUnstructured(clusterRole)
+	type fields struct {
+		Client        client.Client
+		EventRecorder record.EventRecorder
+	}
+	type args struct {
+		rawObj      *unstructured.Unstructured
+		clusterName string
+	}
+	tests := []struct {
+		name    string
+		fields  fields
+		args    args
+		want    *AppliedOverrides
+		want1   *AppliedOverrides
+		wantErr bool
+	}{
+		{
+			name: "test overridePolicies",
+			fields: fields{
+				Client: fake.NewClientBuilder().WithScheme(gclient.NewSchema()).WithObjects(&clusterv1alpha1.Cluster{
+					ObjectMeta: metav1.ObjectMeta{Name: "test1"},
+					Spec: clusterv1alpha1.ClusterSpec{
+						APIEndpoint:                 "https://127.0.0.1",
+						SecretRef:                   &clusterv1alpha1.LocalSecretReference{Namespace: "ns1", Name: "secret1"},
+						InsecureSkipTLSVerification: true,
+						ProxyURL:                    "http://1.1.1.1",
+					},
+				},
+					&corev1.Secret{
+						ObjectMeta: metav1.ObjectMeta{Namespace: "ns1", Name: "secret1"},
+						Data:       map[string][]byte{clusterv1alpha1.SecretTokenKey: []byte("token")},
+					}, deploymentObj, &policyv1alpha1.OverridePolicy{
```

Comment:
The secret and deployment resource seems to no need to be set here.

Reply:
yes

Reply:
If so, just remove it will be ok.